docs/extend/index.md

```diff
@@ -213,7 +213,7 @@ $ git clone https://github.com/vieux/docker-volume-sshfs
 $ cd docker-volume-sshfs
 $ docker build -t rootfs .
 $ id=$(docker create rootfs true) # id was cd851ce43a403 when the image was created
-$ sudo mkdir -p /var/lib/docker/plugins/$id/rootfs
+$ sudo mkdir -p /var/lib/docker/plugins/$id/rootfs/run/docker/plugins
```

Comment:
We dont want to create anything under `rootfs`, it should be the plugins' responsibility. The comment can be clarified. Do you mind updating that?